console/src/log/migrations/v0.ts

```diff
@@ -0,0 +1,32 @@
+import { channel, TimeSpan } from "@synnaxlabs/client";
+import { z } from "zod";
+
+export const stateZ = z.object({
+  key: z.string(),
+  version: z.literal("0.0.0"),
+  retention: z.number(),
+  channels: channel.keyZ.array(),
+  remoteCreated: z.boolean(),
+});
+
+export type State = z.input<typeof stateZ>;
+
+export const ZERO_STATE: State = {
+  key: "",
+  version: "0.0.0",
+  retention: TimeSpan.hours(1).seconds,
```

Comment:
It might be worth changing the `StreamChannelData` class to allow for holding a certain size of data rather than a certain amount of time, but that is definitely more involved.

Reply:
The `StreamChannelData` class is also used to identify the buffering window in the line plot. I think that having a certain retention timespan is better than an amount of data.

Reply:
I can make the retention time span on this longer.